src/main/resources/templates/fragments/footer.html

```diff
@@ -5,10 +5,11 @@
   <a class="text--small font-white spacing-below-15" target="_blank" rel="noopener noreferrer" href="/privacy"
      th:text="#{footer.privacy}"></a>
   <th:block th:if="${#arrays.contains(@environment.getActiveProfiles(), 'demo') || #arrays.contains(@environment.getActiveProfiles(), 'dev')}">
-    <br>Download links (demo-only):
+    <br/>Download links (demo-only):
```

Comment:
Leaving this comment here, though the scope/work of this is outside this PR. 
We should move this to another fragment that only gets included in the "dev" environment.  Or "demo". 
The risks of sharing private data are REALLY high if we forget to take this out - so it might be good to protect against that right now.